apps/frontend/src/components/LobbyPage.tsx

```diff
@@ -35,36 +35,65 @@ function LobbyPage() {
   });
 
   return (
-    <Box>
-      <Button variant="contained" onClick={() => navigate('../difficulty-select')}>
-        Back to Difficulty Select Page;
-      </Button>
+    <Box
+      display={'flex'}
+      flexDirection={'row'}
+      width={'90%'}
+      height={'60vh'}
+      alignItems={'center'}
+      fontFamily={'Arimo'}
+      borderRadius={'10px'}
+      padding={'5%'}
+      style={{ backgroundColor: 'white' }}
+    >
+      <Box display={'flex'} flexDirection={'column'} padding={'5px'} width={'35%'} height={'100%'}>
+        <Box display={'flex'} flexDirection={'column'} width={'85%'} height={'90%'}>
+          <h3 style={{ fontFamily: 'Arimo' }}>Chat</h3>
+          <List sx={{ height: '90%', overflowY: 'auto' }}>
+            {allChatMessages.map((chatMessage, i) => {
+              return (
+                <ListItem key={i} style={{ paddingLeft: '2px' }}>
+                  {chatMessage}
+                </ListItem>
+              );
+            })}
+          </List>
+          <Input
```

Comment:
add `onKeyPress` event so that message is sent when user hits enter on keyboard
https://stackoverflow.com/a/69908211